events.html

```diff
@@ -125,6 +107,12 @@ <h3 class="text-center wdwd">Want to collaborate with us?</h3>
                 <div class="d-flex justify-content-start logo">
                     <div><img src="assets/events/acm_logo_tablet 1.png"></div>
                     <div><img src="assets/events/logo-girlsxtech 2.png"></div>
+                    <div class="newlogo">
+                    <div><img src="assets/events/IIClogo.png"></div></div>
```

Comment:
Using relative path everywhere is better practice, there are lesser chances of your code breaker. So try using relative paths in all src tags. It goes something like `./assets/events/imagename.png`